anitya/admin.py

```diff
@@ -464,3 +464,194 @@ def set_flag_state(flag_id, state):
     return flask.redirect(
         flask.url_for('anitya_ui.browse_flags')
     )
+
+
+@ui_blueprint.route('/users', methods=['GET'])
+@login_required
+def browse_users():
+
+    if not is_admin():
```

Comment:
It would be nice to refactor this code in a decorator, similar to how `@login_required` works. But this should be done in another PR.

Reply:
This could be nice. I will add it to my TODO list.